core/storage/suggestion/gae_models.py

```diff
@@ -482,10 +482,10 @@ def get_in_review_translation_suggestions(
             language_codes: list(str). The list of language codes.
 
         Returns:
-            list(SuggestionModel). A list of suggestions which are in review,
-            but not created by the given user.
+            list(SuggestionModel). A list of translation suggestions which
+            are in review, for the specified language codes but
```

Comment:
Optional nit: add comma after "codes"

Reply:
Sounds good, if and when I will push another commit, I will add this too.